lua/ulx/xgui/settings/cl_urs_gui.lua

```diff
@@ -1 +1,183 @@
		else
			LocalPlayer():ChatPrint( "Missing Argument(s) ~ Please fill in all text boxes." )
		end
	else
		if urs.arg1:GetValue() and urs.arg2:GetValue() then
			LocalPlayer():ConCommand( "ulx loadoutadd \"".. urs.arg1:GetValue() .."\" ".. urs.arg2:GetValue() )
		else
			LocalPlayer():ChatPrint( "Missing Argument(s) ~ Please fill in all text boxes." )
		end
	end
end

--------------------------------------------------------------------------------------------------------------------------------------------
function URSRestrictionProcess( t )
	urs.itemlist:Clear()
	if urs.restrictionlist:GetSelectedLine() then
		if urs.restrictionlist:GetSelected()[1]:GetValue(1) == "Restrictions" then
			if urs.typelist:GetSelectedLine() then
				for item, items in pairs( xgui.data.URSRestrictions[ urs.typelist:GetSelected()[1]:GetValue(1) ] ) do
					for group, groups in pairs( items ) do
						urs.itemlist:AddLine( item, groups )
					end
				end
			end
		end
	end
end
xgui.hookEvent( "URSRestrictions", "process", URSRestrictionProcess )

function URSLimitsProcess( t )
	urs.itemlist:Clear()
	if urs.restrictionlist:GetSelectedLine() then
		if urs.restrictionlist:GetSelected()[1]:GetValue(1) == "Limits" then
			if urs.typelist:GetSelectedLine() then
				for group, groups in pairs( xgui.data.URSLimits[ urs.typelist:GetSelected()[1]:GetValue(1) ] ) do
					urs.itemlist:AddLine( group, groups )
				end
			end
		end
	end
end
xgui.hookEvent( "URSLimits", "process", URSLimitsProcess )

function URSLoadoutsProcess( t )
	urs.itemlist:Clear()
	if urs.restrictionlist:GetSelectedLine() then
		if urs.restrictionlist:GetSelected()[1]:GetValue(1) == "Loadouts" then
			local selected
			if urs.typelist:GetSelectedLine() then
				selected = urs.typelist:GetSelected()[1]:GetValue(1) 
				if xgui.data.URSLoadouts[ selected ] then
					for weapon, weapons in pairs( xgui.data.URSLoadouts[ selected ] ) do
						urs.itemlist:AddLine( weapons )
					end
				end
			end
			urs.typelist:Clear()
			for group, groups in pairs( xgui.data.URSLoadouts ) do
				urs.typelist:AddLine( group )
			end
			if selected then
				if urs.typelist["Lines"] then
					for line, lines in pairs( urs.typelist["Lines"] ) do
						if lines:GetValue(1) == selected then lines:SetSelected( true ) end
					end
				end
			end
		end
	end
end
xgui.hookEvent( "URSLoadouts", "process", URSLoadoutsProcess )

xgui.addSettingModule( "URS", urs.back, "icon16/shield.png" )
\ No newline at end of file
+xgui.prepareDataType( "URSRestrictions" )
+xgui.prepareDataType( "URSLimits" )
+
+urs = {}
+urs.removers = {}
+urs.weapons2 = {"weapon_para","weapon_crowbar","weapon_stunstick","weapon_physcannon","weapon_physgun","weapon_pistol","weapon_357","weapon_smg1","weapon_ar2","weapon_shotgun","weapon_crossbow",  "weapon_frag", "weapon_rpg", "weapon_slam", "weapon_bugbait", "item_ml_grenade", "item_ar2_grenade", "item_ammo_ar2_altfire", "gmod_camera", "gmod_tool"}
```

Comment:
```suggestion
```
I think this can be removed?

Reply:
urs.weapons2 can be removed
![image](https://user-images.githubusercontent.com/69946827/127752452-d7771597-1833-4e62-b3c1-6c2e12699a05.png)

Reply:
Removed